Update benchmark_ci netcoreapp5.0 TFM to target 5.0 release

When you use this command
`py ./scripts/benchmarks_ci.py -f netcoreapp5.0 --filter System.Memory.ReadOnlySpan.IndexOfString`, the automation downloads a build from the 5.0.1xx-rc2 branch, this should be from the release/5.0.1xx branch.

@DrewScoggins have you made some progress on this? if you are too busy I can offer a PR.

I've just tried to run some benchmarks for 5.0 vs 6.0 to describe my recent improvements in MemoryCache, but I was unable to get it working:

```log
PS C:\Projects\performance> git clean -xfd
Removing artifacts/
Removing src/benchmarks/micro/.vs/
Removing tools/
PS C:\Projects\performance> py .\scripts\benchmarks_ci.py -f netcoreapp5.0 net6.0 --filter *MemoryCache*
[2021/01/14 17:11:01][INFO] ----------------------------------------------
[2021/01/14 17:11:01][INFO] Initializing logger 2021-01-14 17:11:01.326712
[2021/01/14 17:11:01][INFO] ----------------------------------------------
[2021/01/14 17:11:01][INFO] Installing tools.
[2021/01/14 17:11:01][INFO] ----------------------
[2021/01/14 17:11:01][INFO] Downloading DotNet Cli
[2021/01/14 17:11:01][INFO] ----------------------
[2021/01/14 17:11:01][INFO] DotNet Install Path: 'C:\Projects\performance\tools\dotnet\x64'
[2021/01/14 17:11:01][INFO] Downloading https://dot.net/v1/dotnet-install.ps1
[2021/01/14 17:11:01][INFO] $ pushd "C:\Projects\performance"
[2021/01/14 17:11:01][INFO] $ powershell.exe -NoProfile -ExecutionPolicy Bypass C:\Projects\performance\tools\dotnet\x64\dotnet-install.ps1 -InstallDir C:\Projects\performance\tools\dotnet\x64 -Architecture x64 -Channel master
[2021/01/14 17:11:02][INFO] dotnet-install: Note that the intended use of this script is for Continuous Integration (CI) scenarios, where:
[2021/01/14 17:11:02][INFO] dotnet-install: - The SDK needs to be installed without user interaction and without admin rights.
[2021/01/14 17:11:02][INFO] dotnet-install: - The SDK installation doesn't need to persist across multiple CI runs.
[2021/01/14 17:11:02][INFO] dotnet-install: To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.
[2021/01/14 17:11:02][INFO]
[2021/01/14 17:11:05][INFO] dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/6.0.100-alpha.1.21064.3/dotnet-sdk-6.0.100-alpha.1.21064.3-win-x64.zip
[2021/01/14 17:13:36][INFO] dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Sdk/6.0.100-alpha.1.21064.3/dotnet-sdk-6.0.100-alpha.1.21064.3-win-x64.zip
[2021/01/14 17:13:52][INFO] dotnet-install: Adding to current process PATH: "C:\Projects\performance\tools\dotnet\x64\". Note: This change will not be visible if PowerShell was run as a child process.
[2021/01/14 17:13:52][INFO] dotnet-install: Note that the script does not resolve dependencies during installation.
[2021/01/14 17:13:52][INFO] dotnet-install: To check the list of dependencies, go to https://docs.microsoft.com/dotnet/core/install/windows#dependencies[2021/01/14 17:13:52][INFO] dotnet-install: Installation finished
[2021/01/14 17:13:52][INFO] $ popd
[2021/01/14 17:13:52][INFO] $ pushd "C:\Projects\performance"
[2021/01/14 17:13:52][INFO] $ powershell.exe -NoProfile -ExecutionPolicy Bypass C:\Projects\performance\tools\dotnet\x64\dotnet-install.ps1 -InstallDir C:\Projects\performance\tools\dotnet\x64 -Architecture x64 -Channel release/5.0.1xx-rc2
[2021/01/14 17:13:53][INFO] dotnet-install: Note that the intended use of this script is for Continuous Integration (CI) scenarios, where:
[2021/01/14 17:13:53][INFO] dotnet-install: - The SDK needs to be installed without user interaction and without admin rights.
[2021/01/14 17:13:53][INFO] dotnet-install: - The SDK installation doesn't need to persist across multiple CI runs.
[2021/01/14 17:13:53][INFO] dotnet-install: To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.
[2021/01/14 17:13:53][INFO]
[2021/01/14 17:13:56][INFO] dotnet-install: Downloading link: https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-rc.2.20480.7/dotnet-sdk-5.0.100-rc.2.20480.7-win-x64.zip
[2021/01/14 17:16:23][INFO] dotnet-install: Extracting zip from https://dotnetcli.azureedge.net/dotnet/Sdk/5.0.100-rc.2.20480.7/dotnet-sdk-5.0.100-rc.2.20480.7-win-x64.zip
[2021/01/14 17:16:38][INFO] dotnet-install: Adding to current process PATH: "C:\Projects\performance\tools\dotnet\x64\". Note: This change will not be visible if PowerShell was run as a child process.
[2021/01/14 17:16:38][INFO] dotnet-install: Note that the script does not resolve dependencies during installation.
[2021/01/14 17:16:38][INFO] dotnet-install: To check the list of dependencies, go to https://docs.microsoft.com/dotnet/core/install/windows#dependencies[2021/01/14 17:16:38][INFO] dotnet-install: Installation finished
[2021/01/14 17:16:38][INFO] $ popd
[2021/01/14 17:16:38][INFO] $ dotnet --info
[2021/01/14 17:16:39][INFO] .NET SDK (reflecting any global.json):
[2021/01/14 17:16:39][INFO]  Version:   6.0.100-alpha.1.21064.3
[2021/01/14 17:16:39][INFO]  Commit:    3e392d143b
[2021/01/14 17:16:39][INFO]
[2021/01/14 17:16:39][INFO] Runtime Environment:
[2021/01/14 17:16:39][INFO]  OS Name:     Windows
[2021/01/14 17:16:39][INFO]  OS Version:  10.0.18363
[2021/01/14 17:16:39][INFO]  OS Platform: Windows
[2021/01/14 17:16:39][INFO]  RID:         win10-x64
[2021/01/14 17:16:39][INFO]  Base Path:   C:\Projects\performance\tools\dotnet\x64\sdk\6.0.100-alpha.1.21064.3\
[2021/01/14 17:16:39][INFO]
[2021/01/14 17:16:39][INFO] Host (useful for support):
[2021/01/14 17:16:39][INFO]   Version: 6.0.0-alpha.1.21063.3
[2021/01/14 17:16:39][INFO]   Commit:  e464963969
[2021/01/14 17:16:39][INFO]
[2021/01/14 17:16:39][INFO] .NET SDKs installed:
[2021/01/14 17:16:39][INFO]   5.0.100-rc.2.20480.7 [C:\Projects\performance\tools\dotnet\x64\sdk]
[2021/01/14 17:16:39][INFO]   6.0.100-alpha.1.21064.3 [C:\Projects\performance\tools\dotnet\x64\sdk]
[2021/01/14 17:16:39][INFO]
[2021/01/14 17:16:39][INFO] .NET runtimes installed:
[2021/01/14 17:16:39][INFO]   Microsoft.AspNetCore.App 5.0.0-rc.2.20475.17 [C:\Projects\performance\tools\dotnet\x64\shared\Microsoft.AspNetCore.App]
[2021/01/14 17:16:39][INFO]   Microsoft.AspNetCore.App 6.0.0-alpha.1.21063.28 [C:\Projects\performance\tools\dotnet\x64\shared\Microsoft.AspNetCore.App][2021/01/14 17:16:39][INFO]   Microsoft.NETCore.App 5.0.0-rc.2.20475.5 [C:\Projects\performance\tools\dotnet\x64\shared\Microsoft.NETCore.App]
[2021/01/14 17:16:39][INFO]   Microsoft.NETCore.App 6.0.0-alpha.1.21063.3 [C:\Projects\performance\tools\dotnet\x64\shared\Microsoft.NETCore.App]
[2021/01/14 17:16:39][INFO]   Microsoft.WindowsDesktop.App 5.0.0-rc.2.20475.6 [C:\Projects\performance\tools\dotnet\x64\shared\Microsoft.WindowsDesktop.App]
[2021/01/14 17:16:39][INFO]   Microsoft.WindowsDesktop.App 6.0.0-alpha.1.21062.4 [C:\Projects\performance\tools\dotnet\x64\shared\Microsoft.WindowsDesktop.App]
[2021/01/14 17:16:39][INFO]
[2021/01/14 17:16:39][INFO] To install additional .NET runtimes or SDKs:
[2021/01/14 17:16:39][INFO]   https://aka.ms/dotnet-download
[2021/01/14 17:16:39][INFO] -------------------------------
[2021/01/14 17:16:39][INFO] Restoring .NET micro benchmarks
[2021/01/14 17:16:39][INFO] -------------------------------
[2021/01/14 17:16:39][INFO] $ pushd "C:\Projects\performance\src\benchmarks\micro"
[2021/01/14 17:16:39][INFO] $ dotnet restore C:\Projects\performance\src\benchmarks\micro\MicroBenchmarks.csproj --packages C:\Projects\performance\artifacts\packages
[2021/01/14 17:16:43][INFO]   Determining projects to restore...
[2021/01/14 17:16:51][INFO]   Restored C:\Projects\performance\src\tools\Reporting\Reporting\Reporting.csproj (in 7.26 sec).
[2021/01/14 17:17:02][INFO]   Restored C:\Projects\performance\src\harness\BenchmarkDotNet.Extensions\BenchmarkDotNet.Extensions.csproj (in 16.4 sec).
[2021/01/14 17:17:03][INFO]   Restored C:\Projects\performance\src\benchmarks\micro\MicroBenchmarks.csproj (in 18.59 sec).
[2021/01/14 17:17:03][INFO] $ popd
[2021/01/14 17:17:03][INFO] ---------------------------------------------------------
[2021/01/14 17:17:03][INFO] Building .NET micro benchmarks for 'net6.0 netcoreapp5.0'
[2021/01/14 17:17:03][INFO] ---------------------------------------------------------
[2021/01/14 17:17:03][INFO] $ pushd "C:\Projects\performance\src\benchmarks\micro"
[2021/01/14 17:17:03][INFO] $ dotnet build C:\Projects\performance\src\benchmarks\micro\MicroBenchmarks.csproj --configuration Release --framework net6.0 --no-restore /p:NuGetPackageRoot=C:\Projects\performance\artifacts\packages
[2021/01/14 17:17:08][INFO] Microsoft (R) Build Engine version 16.9.0-preview-21063-04+27981f15c for .NET
[2021/01/14 17:17:08][INFO] Copyright (C) Microsoft Corporation. All rights reserved.
[2021/01/14 17:17:08][INFO]
[2021/01/14 17:17:11][INFO]   Reporting -> C:\Projects\performance\artifacts\bin\Reporting\Release\netstandard2.0\Reporting.dll
[2021/01/14 17:17:12][INFO]   BenchmarkDotNet.Extensions -> C:\Projects\performance\artifacts\bin\BenchmarkDotNet.Extensions\Release\netstandard2.0\BenchmarkDotNet.Extensions.dll
[2021/01/14 17:17:15][INFO]   MicroBenchmarks -> C:\Projects\performance\artifacts\bin\MicroBenchmarks\Release\net6.0\MicroBenchmarks.dll
[2021/01/14 17:17:15][INFO]
[2021/01/14 17:17:15][INFO] Build succeeded.
[2021/01/14 17:17:15][INFO]     0 Warning(s)
[2021/01/14 17:17:15][INFO]     0 Error(s)
[2021/01/14 17:17:15][INFO]
[2021/01/14 17:17:15][INFO] Time Elapsed 00:00:07.11
[2021/01/14 17:17:15][INFO] $ popd
[2021/01/14 17:17:15][INFO] $ pushd "C:\Projects\performance\src\benchmarks\micro"
[2021/01/14 17:17:15][INFO] $ dotnet build C:\Projects\performance\src\benchmarks\micro\MicroBenchmarks.csproj --configuration Release --framework netcoreapp5.0 --no-restore /p:NuGetPackageRoot=C:\Projects\performance\artifacts\packages
[2021/01/14 17:17:21][INFO] Microsoft (R) Build Engine version 16.9.0-preview-21063-04+27981f15c for .NET
[2021/01/14 17:17:21][INFO] Copyright (C) Microsoft Corporation. All rights reserved.
[2021/01/14 17:17:21][INFO]
[2021/01/14 17:17:22][INFO]   Reporting -> C:\Projects\performance\artifacts\bin\Reporting\Release\netstandard2.0\Reporting.dll
[2021/01/14 17:17:22][INFO]   BenchmarkDotNet.Extensions -> C:\Projects\performance\artifacts\bin\BenchmarkDotNet.Extensions\Release\netstandard2.0\BenchmarkDotNet.Extensions.dll
[2021/01/14 17:17:25][INFO]   MicroBenchmarks -> C:\Projects\performance\artifacts\bin\MicroBenchmarks\Release\netcoreapp5.0\MicroBenchmarks.dll
[2021/01/14 17:17:25][INFO]
[2021/01/14 17:17:25][INFO] Build succeeded.
[2021/01/14 17:17:25][INFO]     0 Warning(s)
[2021/01/14 17:17:25][INFO]     0 Error(s)
[2021/01/14 17:17:25][INFO]
[2021/01/14 17:17:25][INFO] Time Elapsed 00:00:04.14
[2021/01/14 17:17:25][INFO] $ popd
[2021/01/14 17:17:25][INFO] -------------------------------------------------
[2021/01/14 17:17:25][INFO] Running .NET micro benchmarks for 'netcoreapp5.0'
[2021/01/14 17:17:25][INFO] -------------------------------------------------
[2021/01/14 17:17:25][INFO] --------------------------------------------------
[2021/01/14 17:17:25][INFO] Dumping COMPlus environment:
[2021/01/14 17:17:25][INFO] --------------------------------------------------
[2021/01/14 17:17:25][INFO] $ pushd "C:\Projects\performance\src\benchmarks\micro"
[2021/01/14 17:17:25][INFO] $ dotnet run --project C:\Projects\performance\src\benchmarks\micro\MicroBenchmarks.csproj --configuration Release --framework netcoreapp5.0 --no-restore --no-build -- --filter *MemoryCache* --packages C:\Projects\performance\artifacts\packages
[2021/01/14 17:17:29][INFO] It was not possible to find any compatible framework version
[2021/01/14 17:17:29][INFO] The framework 'Microsoft.NETCore.App', version '5.0.0' was not found.
[2021/01/14 17:17:29][INFO]   - The following frameworks were found:
[2021/01/14 17:17:29][INFO]       5.0.0-rc.2.20475.5 at [C:\Projects\performance\tools\dotnet\x64\shared\Microsoft.NETCore.App]
[2021/01/14 17:17:29][INFO]       6.0.0-alpha.1.21063.3 at [C:\Projects\performance\tools\dotnet\x64\shared\Microsoft.NETCore.App]
[2021/01/14 17:17:29][INFO]
[2021/01/14 17:17:29][INFO] You can resolve the problem by installing the specified framework and/or SDK.
[2021/01/14 17:17:29][INFO]
[2021/01/14 17:17:29][INFO] The specified framework can be found at:
[2021/01/14 17:17:29][INFO]   - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=5.0.0&arch=x64&rid=win10-x64
[2021/01/14 17:17:29][INFO] $ popd
[2021/01/14 17:17:29][ERROR] Process exited with status 2147516566
Traceback (most recent call last):
  File ".\scripts\benchmarks_ci.py", line 247, in <module>
    __main(sys.argv[1:])
  File ".\scripts\benchmarks_ci.py", line 230, in __main
    args
  File "C:\Projects\performance\scripts\micro_benchmarks.py", line 310, in run
    *run_args
  File "C:\Projects\performance\scripts\dotnet.py", line 450, in run
    self.working_directory)
  File "C:\Projects\performance\scripts\performance\common.py", line 200, in run
    returncode, quoted_cmdline)
subprocess.CalledProcessError: Command '$ dotnet run --project C:\Projects\performance\src\benchmarks\micro\MicroBenchmarks.csproj --configuration Release --framework netcoreapp5.0 --no-restore --no-build -- --filter *MemoryCache* --packages C:\Projects\performance\artifacts\packages' returned non-zero exit status 2147516566.
```

--- a/scripts/channel_map.py
+++ b/scripts/channel_map.py
@@ -6,6 +6,10 @@ class ChannelMap():
             'tfm': 'net6.0',
             'branch': 'master'
         },
+        '5.0':{
+            'tfm': 'netcoreapp5.0',
+            'branch': 'release/5.0'
+        },
         'release/5.0.1xx-rc2':{
             'tfm': 'netcoreapp5.0',
             'branch': 'release/5.0.1xx-rc2'
@@ -13,10 +17,6 @@ class ChannelMap():
         'release/5.0.1xx':{
             'tfm': 'netcoreapp5.0',
             'branch': 'release/5.0.1xx'
-        },
-        '5.0':{
-            'tfm': 'netcoreapp5.0',
-            'branch': 'release/5.0'
         },
         'release/3.1.3xx':{
             'tfm': 'netcoreapp3.1',

